### ISSUE TITLE
Change the game options

### ISSUE THREAD

**Issue by camembert7** [redmcg/wine-mono#3]
The Elite Dangerous launcher is fine to launch the game, but I would like to change the game language from my native one to English (for consistency purpose) and clicking options in the launcher does nothing. This is not critical, but definitively nice to have!
Thanks !! And congrats for your amazing work.

**Comment by l33tlinuxh4x0r** [comment]
I can confirm that this button doesn't work. :/

**Comment by Neutrovertido** [comment]
I can also confirm said button doesn't work, but you can change the language in one of the configuration .xml files

**Comment by Neutrovertido** [comment]
To elaborate more:
`~/.steam/debian-installation/steamapps/compatdata/359320/pfx/drive_c/users/steamuser/Local Settings/Application Data/Frontier Developments/Elite Dangerous/Options/Player/`

Edit the file `Custom.misc` with the following lines:
```
<Language Value="English" />
<LanguageOverrideActive Value="1" />
```

The launcher won't change the language but the game will.
Hope it was of help!

**Comment by camembert7** [comment]
Thank you, I was looking for the file. Works like a charm.

**Comment by arne-fuchs** [comment]
I tried to change the language values in all CostumXXX.misc files but it doesn't have any impact.
Maybe I am writing the wrong language in the value. Is there maybe a list of possible languages?

**Comment by redmcg** [comment]
This appears to be fixed in wine-mono 6.4. This is currently available in experimental-6.3-20210921 (the latest 6.3 Proton Experimental). I'll close - but please re-open if this is not the case.
